pallets/staking/src/lib.rs

```diff
@@ -1121,7 +1129,25 @@ pub enum Forcing {
 
 impl Default for Forcing {
     fn default() -> Self {
-        Forcing::NotForcing
+        Self::NotForcing
+    }
+}
+
+/// Switch that allows changing slashing status.
```

Comment:
The comment doesn't explain what the slashing status is. Also if would be great to explain the latching behaviour, in other words that the value doesn't go back to `Off`.